lib/unionlabs/src/aptos/block_info.rs

```diff
@@ -32,9 +35,9 @@ pub struct BlockInfo {
     /// The consensus protocol is executed in rounds, which monotonically increase per epoch.
     pub round: Round,
     /// The identifier (hash) of the block.
-    pub id: H256,
+    pub id: Hash<32, HexUnprefixed>,
     /// The accumulator root hash after executing this block.
-    pub executed_state_id: H256,
+    pub executed_state_id: Hash<32, HexUnprefixed>,
```

Comment:
NOTE: replace these with the proper H256 type alias when rebasing #2984 once this is merged